pkg/kubectl/cmd/convert/convert.go

```diff
@@ -60,8 +60,8 @@ var (
 		kubectl convert -f . | kubectl create -f -`))
 )
 
-// ConvertOptions have the data required to perform the convert operation
-type ConvertOptions struct {
```

Comment:
in general, exported Options structs for kubectl commands are the convention to allow them to be reused. that seems better than unexporting

Reply:
Ooh gotcha - thanks for that context. I wasn't aware of that convention.